### DISCUSSION TITLE
Mechanism for managing persistent dependencies

### DISCUSSION THREAD

**Discussion by bryanrsmith** [vercel/turborepo#3475] [Ideas]
I'd love to have a way to manage dependent tasks that are persistent. e.g., 

* integration tests that need to start a local server to test against.
* connect to a planetscale database with the pscale cli before starting a dev task for a project that depends on that database connection.

https://www.npmjs.com/package/start-server-and-test helps manage this, but it makes me break out of turbo for task dependencies. It would be cool to have similar functionality built in to turbo.

**Comment by connorjs** [comment]
If you author npm run scripts, you should be able to execute them through turbo, with whatever `dependsOn` (or other) configuration you desire. https://turbo.build/repo/docs/reference/configuration
